src/apps/tetris.js

```diff
@@ -0,0 +1,152 @@
+import { useEffect, useRef } from "react";
+import Tetris from "react-tetris";
+import styled from "styled-components";
+import { matchTriggerQueries } from "../lib/utilityApis";
+
+const Wrapper = styled.section`
+  .game-block {
+    margin: 0;
+    padding: 0;
+    width: 1.5em;
+    height: 1.5em;
+    border: 1px solid #ddd;
+  }
+  .piece-i {
+    background-color: #ec858b;
+  }
+  .piece-j {
+    background-color: #f1b598;
+  }
+  .piece-l {
+    background-color: #f8efae;
+  }
+  .piece-o {
+    background-color: #b5a677;
+  }
+  .piece-s {
+    background-color: #816e56;
+  }
+  .piece-t {
+    background-color: #b77c72;
+  }
+  .piece-z {
+    background-color: #e3be58;
+  }
+  .piece-preview {
+    background-color: #eee;
+  }
+`;
+
+const StyledContainer = styled.div`
+  display: grid;
+  grid-template-columns: auto auto auto;
+  background-color: #080909;
+  padding: 10px;
+`;
+
+const StyledItem = styled.div`
+  background-color: #aaaaaa;
+  border: 1px solid rgba(0, 0, 0, 0.8);
+  padding: 10px;
+  font-size: 1rem;
+  text-align: center;
+`;
```

Comment:
I feel like the background colour and thick border are making the light colours of tetris dull. 
```suggestion
const StyledContainer = styled.div`
  display: grid;
  grid-template-columns: auto auto auto;
  padding: 10px;
`;

const StyledItem = styled.div`
  border: 1px solid #ccc;
  padding: 10px;
  font-size: 1rem;
  text-align: center;
`;
```

What do you think about this? Transparent background but light border to separate all the divisions.

Reply:
This does look way cooler